### ISSUE TITLE
Setting X-Requested-With header in V3

### ISSUE THREAD

**Issue by gnito-org** [afcapel/stimulus-autocomplete#93]
I upgraded to Symfony 6, and in the process had to upgrade to ``stimulus-autocomplete:^3.0.0-rc.6``.

The behavior I'm observing is that stimulus-autocomplete is doing an Ajax call, but it is not setting the ``X-Requested-With`` header to ``XMLHttpRequest``. 

In ``stimulus-autocomplete:^2.0`` this worked perfectly.

**Comment by gnito-org** [comment]
Looking at how V2 was written, should [this line in V3](https://github.com/afcapel/stimulus-autocomplete/blob/c69fbc25f15560d8c8cb46b925e8efdd021c2416/src/autocomplete.js#L223) perhaps be as follows?

```
const response = await fetch(url, { this.headersForFetch() })
```
instead of:

```
const response = await fetch(url, this.headersForFetch())
```

**Comment by afcapel** [comment]
@gnito-org thanks for the bug report!

Fixed in 579a03308a9aebf056a64d13a9f7a91504661c11, which is available in 3.0.0-rc.7.
